packages/admin-stories/src/docs/components/Selection/Selection.stories.mdx

```diff
@@ -0,0 +1,111 @@
+import { Meta, Story, Canvas, Source } from "@storybook/addon-docs";
+
+<Meta title="Docs/Components/Selection" />
+
+# Selection
+
+In Comet Admin, a _selection_ can be used to manage which item of a list or table a user has currently selected.
+It's most commonly used for keeping track of selections when combining a table and a form.
+
+The two most common use cases for selections are
```

Comment:
This two sentences basically are the same. We could remove the first in favor of the second